guide/matrix-vs-telegram.md

```diff
@@ -28,27 +28,27 @@ Despite recent studies, Telegram's use of its own MTProto protocol remains a deb
 
 | Feature | Telegram | Matrix |
 | ------- | -------- | ------ |
-| **Registration** | Requires phone number. | **Phone number is always optional.** Depending on homeserver, **email may not be required**. |
-| **Username** | Users are identified by phone number or username (if set up, 5~32 alphanumeric characters) to fellow users, and user IDs (around 9~10 digits) for programming purposes. A display name can be added (no limit). | Users are identified by their MXID (eg. `@alice:example.com`), composed of the username (must be ASCII characters, but no length limit) and the server name. A display name can be optionally added (no limit). |
+| **Registration** | Requires phone number. | **Phone number is usually optional.** Depending on homeserver, **email may not be required**. There is usually no human check after registration. |
```

Comment:
See above.

Reply:
SSO and 3PID makes it possible to enforce Phone numbers on matrix tho its almost purely theoretical at this point. The point is more that its completely possible. 

```suggestion
| **Registration** | Requires phone number. | **Phone number is usually optional.** Depending on homeserver, **email is commonly required**. There is usually no human check after registration. |
```